grails-app/conf/au/org/emii/aatams/VisibilityControlFilters.groovy

```diff
@@ -36,15 +36,16 @@ class VisibilityControlFilters
         {
             after = { model ->
 
-                def instanceName = "${controllerName}Instance"
-                def instance = model[instanceName]
+                if (model) {
```

Comment:
Do you think it would show the intent better then if we changed the filter method from `genericNotList` to ??? which handles only `show` and `edit` actions?